File: bot.js
const { Client, LocalAuth } = require('whatsapp-web.js');
const qrcode = require('qrcode-terminal');

const client = new Client({
    authStrategy: new LocalAuth()
});

client.on('qr', qr => {
    console.log('?? Escanea este QR con WhatsApp Web:');
    qrcode.generate(qr, { small: true });
});

client.on('ready', () => {
    console.log('? Bot de WhatsApp conectado y listo.');
});

client.on('message', async message => {
    console.log(`?? Mensaje recibido: ${message.body}`);

    // Palabras clave para reenviar
    const keywords = ["TV", "controles", "luces"];

    if (keywords.some(word => message.body.toLowerCase().includes(word.toLowerCase()))) {
        const chat = await message.getChat();
        if (chat.isGroup) {
            console.log('?? Reenviando mensaje...');
            await message.forward('GRUPO_DESTINO_ID'); // Reemplaza con el ID real del grupo destino
        }
    }
});

client.initialize();

File: bot.py
from twilio.rest import Client

# Tus credenciales de Twilio
account_sid = 'your_account_sid'
auth_token = 'your_auth_token'
client = Client(account_sid, auth_token)

# Enviar mensaje de WhatsApp
message = client.messages.create(
    body="¡Hola! Este es un mensaje de prueba desde Twilio.",
    from_='whatsapp:+14155238886',  # Tu número de Twilio (con prefijo whatsapp:)
    to='whatsapp:+521XXXXXXXXXX'  # El número de destino (con prefijo whatsapp:)
)

print(message.sid)  # Muestra el SID del